packages/legend-extension-dsl-data-space-studio/src/components/FormTextEditor.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useEditorStore } from '@finos/legend-application-studio';
+import { DataSpaceEditorState } from '../stores/DataSpaceEditorState.js';
+import { useRef, useEffect } from 'react';
+import { PanelFormTextField, PanelFormSection } from '@finos/legend-art';
+
+export const FormTextEditor = observer(() => {
```

Comment:
Lets rename here to match other editor, so here it would be DataSpaceEditor or DataSpaceElementEditor

Reply:
Done